From fc7b67b7a0c5e7333f7f836940cf921e8a3bd711 Mon Sep 17 00:00:00 2001
From: Subramanian Srinivasan <subrsrin@codeaurora.org>
Date: Mon, 29 Apr 2019 09:38:19 +0200
Subject: [PATCH 3/3] Fix handling of BLE create connection when all PHYs are
 set When all PHYs are set during BLE create connection, there is an assert
 error seen in BT stack. This change fixes this issue.

Change-Id: Ieffb4f93f34547699f05aad6028223c096595943
Signed-off-by: DennySPB <dennyspb@gmail.com>
---
 stack/btm/btm_ble_bgconn.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stack/btm/btm_ble_bgconn.cc b/stack/btm/btm_ble_bgconn.cc
index 952e9530a..d6fc45172 100644
--- a/stack/btm/btm_ble_bgconn.cc
+++ b/stack/btm/btm_ble_bgconn.cc
@@ -372,7 +372,7 @@ void btm_send_hci_create_connection(
         std::bitset<std::numeric_limits<uint8_t>::digits>(initiating_phys)
             .count();
 
-    LOG_ASSERT(phy_cnt < 3) << "More than three phys provided";
+    LOG_ASSERT(phy_cnt < 4) << "More than three phys provided";
     // TODO(jpawlowski): tune parameters for different transports
     for (int i = 0; i < phy_cnt; i++) {
       phy_cfg[i].scan_int = scan_int;
-- 
2.20.1

